client/landing/stepper/hooks/use-prepare-site-for-migration.ts

```diff
@@ -199,50 +203,3 @@ export const usePrepareSiteForMigrationWithMigrateGuru = ( siteId?: number ) =>
 		migrationKey: migrationKey ?? null,
 	};
```

Comment:
While I am a _huge_ fan of this cleanup, you haven't documented these changes as part of the PR description or the main testing instructions. Could you please update the docs or the code so they're aligned?

Reply:
This branch was based on https://github.com/Automattic/wp-calypso/pull/95116 and for some reason got messed up once that PR got merged. It's cleaned up now. Sorry for the confusion.